index.js

```diff
@@ -2787,64 +2787,62 @@
   }
   S.prop = def('prop', {a: [Accessible]}, [$.String, a, b], prop);
 
-  //# get :: Accessible a => TypeRep b -> String -> a -> Maybe b
+  //# get :: Accessible a => (b -> Boolean) -> String -> a -> Maybe c
   //.
-  //. Takes a [type representative](#type-representatives), a property
-  //. name, and an object and returns Just the value of the specified object
-  //. property if it is of the specified type (according to [`is`](#is));
+  //. Takes a predicate, a property name, and an object and returns Just the
+  //. value of the specified object property if it passes the given predicate;
   //. Nothing otherwise.
```

Comment:
We should state that `Nothing` is returned if the property does not exist:

_Takes a predicate, a property name, and an object and returns Just the value of the specified object property if <del>it passes</del> <ins>the property exists and the value satisfies</ins> the given predicate; Nothing otherwise._